### PR TITLE
asdf: add German translation

### PR DESCRIPTION
**Pull request by DeMuu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/asdf.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# asdf
+
+> Verwalte installierte Versionen von verschiednen Paketen.
```
Comment (pixelcmtd) [inline pages.de/common/asdf.md:3]:
```suggestion
> Verwalte installierte Versionen von verschiedenen Paketen.
```

File: pages.de/common/asdf.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Liste alle verfügbaren Plugins auf:
+
+`asdf plugin list all`
+
+- Installiere ein neues Plugin:
+
+`asdf plugin add {{name}}`
+
+- Liste alle verfügbaren Versionen für ein Paket:
```
Comment (pixelcmtd) [inline pages.de/common/asdf.md:15]:
```suggestion
- Liste alle verfügbaren Versionen für ein Paket auf:
```

**Review by kbdharun — COMMENTED** [review]
@DeMuu Thanks for your contribution, Welcome to tldr. Translate the `name` and `version` command arguments inside `{{}}` too.

**Review by pixelcmtd — APPROVED** [review]
LGTM.

@kbdharun They already are :D

**Comment by kbdharun** [comment]
> @kbdharun They already are :D

Oh, my bad 😄

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**